src/index.js

```diff
@@ -97,7 +97,7 @@ const logIn = async function(fields) {
     $cgu.attr('content').includes('as_conditions_generales_page')
   ) {
     log('debug', $cgu.attr('content'))
-    throw new Error(errors.USER_ACTION_NEEDED)
+    throw new Error('USER_ACTION_NEEDED.CGU_FORM')
```

Comment:
Do you think it's possible to add a new error to cozy-konnector-libs? This will prevent future problem if the string value change.

Reply:
Yes, I will do this, but I wanted to publish this fix to help us know the different error cases for this connector

Reply:
Here it is : https://github.com/konnectors/libs/pull/284 :-)